ostree/ostree_dest.go

```diff
@@ -11,12 +11,15 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/containers/image/manifest"
 	"github.com/containers/image/types"
 	"github.com/containers/storage/pkg/archive"
 	"github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
+
+	"github.com/ostreedev/ostree-go/pkg/otbuiltin"
```

Comment:
@runcom per the other scattered conversations… I guess it would be nice to have a single “I don’t want to care, give me the no-dependencies subset” switch (as long as that subset is useful to a fair amount of users), instead of a dozen flags. I haven’t yet looked into whether that is possible/easy, I guess it’s now quite urgent.

As far as the effect, I think just making the `alltransports` inclusion conditional would be quite enough?
- We really don’t want to maintain two different `ostree` implementations
- Users who cherry-pick a transport don’t have to care about anything
- Users of `alltransports` would either set up the build environment as necessary, or opt out using the TBD mechanism.